sensor.py

```diff
@@ -39,11 +45,15 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
 
     client = wideq.Client.from_token(refresh_token, region, language)
     dishwashers = []
+    acs = []
 
     for device_id in hass.data[KEY_SMARTTHINQ_DEVICES]:
         device = client.get_device(device_id)
         model = client.model_info(device)
 
+        LOGGER.debug('SMARTTHINQ_DEVICE: %s' % device.type)
+        LOGGER.debug('SMARTTHINQ_DEVICE: %s' % wideq.DeviceType.AC)
```

Comment:
Also perhaps unnecessary now? Or perhaps we could just make the messages more readable with English phrases?